docs/source/release-notes.rst

```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+Version 0.18.1
+______________
+- Engine:
+
+  - SERVICE events for continuously running processes now provides a `service_running`
```

Comment:
Breaking change: SERVICE events for continuously running processes now provides a `service_running` to be used instead of infinite loops.